src/monopoly.js

```diff
@@ -134,6 +140,41 @@ export default class Monopoly {
     return this;
   }
 
+  loadPropertyArray(){
+    this.propertyArray = JSON.parse(fs.readFileSync('properties.JSON'));
+    this.propertyArray.forEach((property) => {
+      switch (property.functionID) {
+        case 1:
```

Comment:
Still suggest replacing functionID with string that represents the property

Then your case would look like:

```
case 'standard': 
    property.landOnFunction = standardProperty; 
    break;
case 'railroad': 
    property.landOnFunction = railroad;
    break;
// etc. etc.
```

Just makes the code clearer.